modules/microshift-custom-ca-cert-rotation.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-cert-rotation_{context}"]
+= Allowing {microshift-short} to start with an expired custom certificate authority
+
+External certificates are not automatically renewed. You must manually rotate your external certificates.
+
+.Procedure
+
+. To allow {microshift-short} to start with an expired custom certificate, use the `--insecure-skip-tls-verify` client mode by adding the following example configuration file to the host:
```

Comment:
this config  doesn't affect Microshift service..
 this is `kubeconfig` configuration that  tells the oc `client` to  behave on  insecure manner and  accept insecure certificates (even if they are expired)